actionkit/api/user.py

```diff
@@ -79,6 +79,9 @@ def user_search(self, query_params={}):
         res = self.client.get('%s/rest/v1/user/' % (self.base_url), params=query_params)
         return {'res': res, 'users': res.json() if res.status_code == 200 else None}
 
+    def direct_request(self, request_string):
+            res = self.client.get('%s%s' % (self.base_url, request_string))
+            return {'res': res, 'users': res.json() if res.status_code == 200 else None}
```

Comment:
If this is always returning users, maybe rename it? Otherwise it should probably return 'body' or 'response' or 'json' or similar.

Reply:
Wait never mind I see this is in the "user" API file.